docs/src/features.md

```diff
@@ -176,17 +230,57 @@ For the connection from the hub to the demand there are associated transmission
 ```math
```

Comment:
Sorry, I cannot seem to add a comment/suggestion to the part that you did not change. Actually I have a suggestion to be put at line 216 in this commit (in #### Producers Capacity Constraints, after the equations for `wind`)

> Since the flow variables `$flow_{(wind,balance),1:2}$` `$flow_{(wind,balance),1:3}$` represent power, the first constraint sets the upper bound of the power for both time step 1 and 2, by assuming an average capacity across these two time steps. The same applies to the other two constraints.

Reply:
I agree that that's needed there. Thanks! I'll add it.